<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>420</width>
      <height>320</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="toolButton_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>96</y>
       <width>54</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="login_condition">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>134</y>
       <width>37</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="settings">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>0</y>
       <width>30</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="setMin">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>30</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>388</x>
       <y>0</y>
       <width>30</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>255</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="username">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>175</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>175</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>255</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="password">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>205</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>自动登录</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>237</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>找回密码</string>
     </property>
    </widget>
    <widget class="QToolButton" name="QRcode">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="register_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="backlabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>420</width>
       <height>320</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>205</y>
       <width>17</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>backlabel</zorder>
    <zorder>toolButton_8</zorder>
    <zorder>settings</zorder>
    <zorder>setMin</zorder>
    <zorder>close_btn</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>username</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit</zorder>
    <zorder>password</zorder>
    <zorder>checkBox</zorder>
    <zorder>checkBox_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>QRcode</zorder>
    <zorder>login</zorder>
    <zorder>register_2</zorder>
    <zorder>label_2</zorder>
    <zorder>login_condition</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
